Makefile.frankfurt

```diff
@@ -170,8 +175,8 @@ requirements-py3.txt:
 build: doc translate chsdi/static/css/extended.min.css rss
 
 
-.PHONY: environ
-environ: guard-ENV_FILE guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD
+.PHONY: config-templates
+config-templates: guard-ENV_FILE guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWORD
```

Comment:
It works for me, and I don't see why it would fail. I mean the `guard-ENV_FILE` make sure that it set before doing the `export $(shell cat $(ENV_FILE))`. I'll update the docu where the manual export of the env file is not needed anymore though.

Reply:
Oh I just saw that at the top of the makefile I include the `ENV_FILE` so the `guard-` for `ENV_FILE`, which has a default value to `.env.default` at the top, is useless I'll remove this guard.